eurobot_loc/launch/pf_launch.launch

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
rename to test_localization.launch
add secondary_robot_localization.launch
add main_robot_localization.launch

Reply:
mb localization.launch

Reply:
pf, lidar